docs/changelog.mdx

```diff
@@ -4,6 +4,33 @@ sidebar_label: 'Changelog'
 
 # Changelog
 
+## Release 0.0.22
+
+_June 18, 2024_
+
+**New Features**
+
+- The SDK now supports recording games using the legacy input system and Unity UI.
+- New data collection format for improved state display within the scenario builder and validation results page.
+- New Bot Segment data format for stitching together various bot types, such as monkey testing bots combined with recordings.
+- Bounding boxes show entity locations within the gameplay session for improved debugging.
+
+## Release 0.0.21
```

Comment:
I guess I never made a release note for the last release, so added that here as well.

Reply:
I think we need to differentiate ClientUI vs SDK in these lines

Reply:
Adjusted this slightly, but wasn't 100% sure which part was the unclear part - let me know if this is better!

Reply:
I think the "bounding boxes" part needs to specify that it's a client UI feature